src/app/submit-item/components/submit-page/submit-page.component.html

```diff
@@ -7,9 +7,9 @@
           <h1>Fall einreichen</h1>
           <div>
             <span class="submit-text">
-              Gib hier die Details zu deinem Fall ein. Das kann entweder eine Behauptung sein, oder ein Link. Sobald du auf Fall absenden
-              klickst, prüfen wir ob wir diesen Fall schon bearbeitet haben. Dann erhältst du sofort ein Ergebnis. Falls dein Fall neu ist,
-              werden wir ihn sofort an unser Team von Detektiven weitergeben, die den Fall dann für dich prüfen.
+              Hier kannst du deinen Fall einreichen. Wenn der Fall bereits von unserer Community an Detektiv*innen
+              gelöst wurde, erhältst du das Ergebnis in Form des Vertrauens-Index sofort. Ansonsten übergeben wir den
+              Fall direkt an unsere Detektiv*innen.
```

Comment:
"Community an Detektiv*innen"  sounds a little bit weird.
Would propose "Detektiv-Community" - even if there is no gender specification.